change/@microsoft-teams-js-6a7d3368-138b-40f7-86c3-0d9e78441ff5.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Minor change in helper values",
```

The comment you have here is insufficient; it gets turned patch notes for internal and external developers. Try something like:

```suggestion
  "comment": "Fixed bug where `thirdPartyCloudStorage.getDragAndDropFiles` failed deterministically",
```